Library/Formula/kpcli.rb

```diff
@@ -0,0 +1,95 @@
+class Kpcli < Formula
+  desc "command-line interface to KeePass database files"
+  homepage "http://kpcli.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/kpcli/kpcli-3.0.pl"
+  sha256 "947b7b0485215f72e14bb8936c847abb583253c597f58234650922270259049c"
+
+  bottle :unneeded
```

Comment:
There's a small mountain of resources and a compile process here; we'll need bottles.